sdk/storage/Azure.Storage.DataMovement/src/TransferManager.cs

```diff
@@ -147,11 +152,17 @@ public virtual async Task PauseTransferIfRunningAsync(string transferId, Cancell
         /// If not specified or specified to <see cref="DataTransferState.None"/>,
         /// all transfers will be returned regardless of status.
         /// </param>
+        /// <param name="cancellationToken">
+        /// Optional <see cref="CancellationToken"/> to propagate
+        /// notifications that the operation should be cancelled.
+        /// </param>
         /// <returns></returns>
         public virtual async IAsyncEnumerable<DataTransfer> GetTransfersAsync(
+            [EnumeratorCancellation] CancellationToken cancellationToken = default,
```

Comment:
I actually don't know the correct style choice here and the standard code quality analyzer rule for cancellation token positioning doesn't mention `params` in its docs at all.

I would say the correct choice is to remove params entirely and take `ICollection<DataTransferStatus>` instead.

This is our public api though. So I'd ask @amnguye to chime in since she's been in a lot of api meetings for this.

Reply:
Yeah I think the 'params' makes things a little awkward. I've removed it for now but I'll see what @amnguye says when she comes back

Reply:
Amanda has approved the API changes